client/src/overview/components/SizeSelector.jsx

```diff
@@ -25,10 +25,12 @@ let SizeSelector = ({selectedStyle, setPurchase}) => {
       let skuNumber = selectedStyle.skus[key]
       let buttonSize = skuNumber.size
       let quantity = skuNumber.quantity
-      if (key == 'null' || !key || !quantity) return <button key={i} disabled>{<s>{buttonSize}</s> || 'Out Of Stock'}</button>
+      if (key == 'null' || !key || !quantity) {
+        return <button key={i} disabled>{{buttonSize} || 'Out Of Stock'}</button>
+      }
       return (<button key={key} onClick={changeSku.bind(this, skuNumber)}>{buttonSize}</button>)
```

Comment:
@JeromeMTR is this bind necessary?  I think its redundant since you declared changeSku on line 9 using an arrow function.

Reply:
@crypto-bender I could go with it either two way, an anonymous function or bind, if I dont have it it'll execute infinitely.

Reply:
@crypto-bender Agree! Youre right that function is redundant. Tyty